### DISCUSSION TITLE
PermanentDaysOff property

### DISCUSSION THREAD

**Discussion by kalwinskidawid** [allenporter/pyrainbird#458] [General]
Hey in the unit tests of the schedule you have property `permanentDaysOff`, but it is always set to `0`, and in `data.py` there is no the parameter. Should it be deserialized like `daysOfWeek` ?  Also 'reserved' is only used in test cases

**Comment by allenporter** [comment]
The unit tests have examples from real devices and my impression is they are not using this field in the test responses. I think its just because of the test data.  The code is doing deserialization here https://github.com/allenporter/pyrainbird/blob/c8db8294d39ddab7e0d73d39e045008b665b3dee/pyrainbird/rainbird.py#L61

If you have sample responses from your device setting those fields, we can use them for test cases.

**Reply by kalwinskidawid** [reply]
Thanks, when I'll have any access to RB then I can provide some data. BTW. Do you plan to implement set_schedule in your project? :)
